scripts/sed

```diff
@@ -93,16 +93,16 @@ s/\<[^-]symlink/symbolic link/g
 s/\<([Mm])iddle[- ][Ww]are/\1iddleware/g
 
 # plural
-s/\<([Ff])iles?[- ]?[Ss]ystem(s)?\>/\1ile system\2/g
-s/\<([Tt])ypes?[- ]?[Ss]ystem(s)?\>/\1ype system\2/g
-s/\<([Tt])ypes?[- ]?[Cc]hecker(s)?\>/\1ype checker\2/g
+s/\<^([Ff])iles?[- ]?([Ss])ystem(s)?$\>/\1ile \2ystem\3/g
```

Comment:
What does this mean? Iirc `\<` is start of word and `^` is start of line?

Reply:
Here some overmatching occurred, so that also parts, where parameters for commands are explained, got changed. The "fix" here was of course not correct.

Now I changed the script so that only the plural of the first word gets changed to singular, but the part between the two words stays as it is.

As a general good practice, I would strongly recommend to check all changes done by the script before committing any changed files.